tests/wpunit/Tribe/Tickets/Attendance_RSVP_TotalsTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Tribe\Tickets;
+
+use Tribe\Events\Test\Factories\Event;
+use Tribe\Tickets\Test\Commerce\RSVP\Ticket_Maker as RSVP_Ticket_Maker;
+use Tribe\Tickets\Test\Commerce\Attendee_Maker as Attendee_Maker;
+use Tribe__Tickets__RSVP__Attendance_Totals as Tickets__Attendance;
+
+/**
+ * Test Calculations
+ *
+ * Class Attendance_RSVP_TotalsTest
+ *
+ * @package Tribe\Tickets
+ */
+class Attendance_RSVP_TotalsTest extends \Codeception\TestCase\WPTestCase {
+
+	use RSVP_Ticket_Maker;
+	use Attendee_Maker;
+
+	public function setUp() {
+		// before
+		parent::setUp();
+
+		$this->factory()->event = new Event();
+
+		// let's avoid confirmation emails
+		add_filter( 'tribe_tickets_rsvp_send_mail', '__return_false' );
+
+		add_filter( 'tribe_tickets_user_can_manage_attendees', '__return_true' );
+
+	}
+
+	public function tearDown() {
+		// your tear down methods here
+		// then
+		parent::tearDown();
+	}
+
+	/**
+	 * @test
+	 */
+	public function it_should_count_total_rsvps_correctly() {
+
+		$event_id          = $this->factory()->event->create();
+		$rsvp_id           = $this->create_rsvp_ticket( $event_id );
+		$created_attendees = $this->create_many_attendees_for_ticket( 10, $rsvp_id, $event_id );
+
+		$Tickets__Attendance = new Tickets__Attendance( $event_id );
```

Comment:
We should create a `make_instance()` and  `it_should_be_instanceable` tests for this class. Also, avoid the upper case on the variable names.